pack/valk_encounter.json

```
@@ -54,19 +59,22 @@
 		"quantity": 1,
 		"set_code": "valk_nemesis",
 		"set_position": 3,
+		"text": "Treat attached ally as an [enthralled] minion with a blank text box.\nAttached minion's SCH is equal to its printed THW and it does not take consequential damage.\n<b>When Revealed</b>: Attach to the ally with the highest cost without Beguiled attached. Attached ally engages its controller. Otherwise, this card gains surge.",
```

To obtain the bold format for these keywords I think the text needs to be surrounded by double square brackets (e.g. `[[Keyword]]`)

```suggestion
		"text": "Treat attached ally as an [[Enthralled]] minion with a blank text box.\nAttached minion's SCH is equal to its printed THW and it does not take consequential damage.\n<b>When Revealed</b>: Attach to the ally with the highest cost without Beguiled attached. Attached ally engages its controller. Otherwise, this card gains surge.",
```